fcli-core/fcli-fod/src/main/java/com/fortify/cli/fod/rest/lookup/helper/FoDLookupHelper.java

```diff
@@ -0,0 +1,145 @@
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.fortify.cli.common.output.transform.fields.RenameFieldsTransformer;
+import com.fortify.cli.fod._common.rest.FoDUrls;
+import com.fortify.cli.fod.rest.lookup.cli.mixin.FoDLookupTypeOptions;
+import kong.unirest.GetRequest;
+import kong.unirest.UnirestInstance;
+import lombok.Getter;
+
+import javax.validation.ValidationException;
+import java.util.Iterator;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class FoDLookupHelper {
+    @Getter
+    private static final ObjectMapper objectMapper = new ObjectMapper();
+
+    private static final String[] lookupItems = new String[]{
+            "MobileScanPlatformTypes",
+            "MobileScanFrameworkTypes",
+            "MobileScanEnvironmentTypes",
+            "MobileScanRoleTypes",
+            "MobileScanExternalDeviceTypes",
+            "DynamicScanEnvironmentFacingTypes",
+            "DynamicScanAuthenticationTypes",
+            "TimeZones",
+            "RepeatScheduleTypes",
+            "GeoLocations",
+            "SDLCStatusTypes",
+            "DayOfWeekTypes",
+            "BusinessCriticalityTypes",
+            "ReportTemplateTypes",
+            "AnalysisStatusTypes",
+            "ScanStatusTypes",
+            "ReportFormats",
+            "Roles",
+            "ScanPreferenceTypes",
+            "AuditPreferenceTypes",
+            "EntitlementFrequencyTypes",
+            "ApplicationTypes",
+            "ScanTypes",
+            "AttributeTypes",
+            "AttributeDataTypes",
+            "MultiFactorAuthorizationTypes",
+            "ReportTypes",
+            "ReportStatusTypes",
+            "PassFailReasonTypes",
+            "DynamicScanWebServiceTypes",
+            "VulnerabilitySeverityTypes",
+            "TechnologyTypes",
+            "LanguageLevels",
+            "AuditActionTypes",
+            "NotificationTriggerTypes",
+            "ConcurrentRequestThreadsTypes",
+            "MobileScanAuditPreferenceTypes",
+            "DataExportTypes",
+            "ScanMethodTypes",
+            "StartScanMethodTypes",
+            "AuditTemplateConditionTypes",
+            "OpenSourceScanTypeAuditTemplateFieldTypes",
+            "StaticScanTypeAuditTemplateFieldTypes",
+            "DynamicMobileScanTypeAuditTemplateFieldTypes"
+    };
+
+    public static final JsonNode renameFields(JsonNode record) {
+        return new RenameFieldsTransformer(new String[]{}).transform(record);
+    }
+
+    public static final ArrayNode getLookupTypes() {
+
+        //String[] lookups = Stream.of(FoDLookupType.values()).map(FoDLookupType::name).toArray();
+        List<String> lookups = Stream.of(FoDLookupTypeOptions.FoDLookupType.values())
+                .map(FoDLookupTypeOptions.FoDLookupType::name)
+                .collect(Collectors.toList());
+        System.out.println(lookups);
```

Comment:
I guess this is debugging code; please remove all `System.out.println` calls

Reply:
Yes, I was trying something and forgot to take it out, removed it now.